### DISCUSSION TITLE
Hello, I got this error in previewing after using RC10 to build. May I ask what is the cause

### DISCUSSION THREAD

**Discussion by yufanzheng0723** [nuxt/nuxt#16762] [Questions]
This error occurred in the preview after I used PNPM Run Build
[nuxt] [request error] [unhandled] [500] The requested module 'vue' does not provide an export named 'default'

**Comment by sevillaarvin** [comment]
Can you provide a reproduction repo? My guess is that `vue` was imported without destructuring somewhere in the code.

**Reply by yufanzheng0723** [reply]
Thank you for your idea

**Comment by TheAlexLichter** [comment]
You need to install the deps via `pnpm i --shamefully-hoist` ☺️

**Reply by yufanzheng0723** [reply]
Thanks for your help. I figured out what my problem is

**Reply by TheAlexLichter** [reply]
What whas it @yufanzheng0723? ☺️
